server/backends/chunkstore/chunkstore.go

```diff
@@ -363,12 +385,9 @@ func (w *ChunkstoreWriter) readFromWriteResultChannel() (int, error) {
 	}
 }
 
-func (w *ChunkstoreWriter) GetLastChunkIndex() (uint16, error) {
+func (w *ChunkstoreWriter) GetLastChunkIndex() uint16 {
 	w.Write([]byte{})
```

Comment:
nit: remove this NOP write, or add a comment explaining why it's needed?

Reply:
Commented.